packages/passwordless-auth-construct/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * A placeholder for the code or link in messages.
+ */
+export const codeOrLinkPlaceholder = '##codeOrLink##';
```

Comment:
Question: do we want to allow this to be customized?

Reply:
It is only used internally. This isn't exposed to the customer. The API that the customer uses is

```ts
message: (code: string) => `Your code is ${code}`
```